src/hooks/useAsyncStorage.tsx

```diff
@@ -23,6 +23,29 @@ export function useAsyncStorage(key: string, enabled: boolean = true) {
     [key, value, enabled],
   );
 
+  const mergeItem = useCallback(
```

Comment:
Added this but it's not actually used anywhere in this PR but I know of at least a couple cases that could benefit from this down the line. Needs to be tested.